src/modules/lifecycle_management.json

```diff
@@ -0,0 +1,254 @@
+{
+    "openrpc": "1.2.4",
+    "info": {
+        "title": "LifecycleManagement",
+        "description": "A module for managing application lifecycle.",
+        "version": "0.0.0"
+    },
+    "components": {
```

Comment:
Can we keep methods before components, by convention? it's much easier to start reading a doc with the methods first, IMO